public/app/features/plugins/admin/helpers.ts

```diff
@@ -51,6 +51,8 @@ export function mergeLocalsAndRemotes({
           ? true
           : instancesSet.has(remotePlugin.slug) && catalogPlugin.isInstalled;
         catalogPlugin.isInstalled = instancesSet.has(remotePlugin.slug) || catalogPlugin.isInstalled;
+
+        catalogPlugin.isUninstalling = Boolean(localCounterpart) && !instancesSet.has(remotePlugin.slug);
```

Comment:
I think this name could be a bit misleading in OSS / On-prem (if the plugin is simply installed on-prem, then this should evaluate to `true`, as `!instancesSet.has(remotePlugin.slug)` will be `true`, right?)

Naming it something like `isUninstallingFromRemote()` or `isUninstallingFromInstance()` could be more clear I think. What do you think?